frontend/ui/data/css_tweaks.lua

```diff
@@ -83,6 +133,14 @@ h4, h5, h6 { page-break-after: avoid !important; }
                 title = _("Justify all elements"),
                 description = _("Text justification is the default, but it may be overridden by publisher styles. This will re-enable it for all elements, which may lose centering in some of them."),
                 css = [[* { text-align: justify !important; }]],
+                separator = true,
+            },
+            {
+                id = "headings_align_center",
+                title = _("Center headings"),
+                description = _("Middle align headings."),
```

Comment:
I'm not sure if that's really any clearer than `center align`, if not actually confusing? I think the implicit "align" part is the important one. Middle align means vertical "centering" not just in CSS but also in Excel/Calc etc.